### DISCUSSION TITLE
Repository Insights Contributors page does not show my commit

### DISCUSSION THREAD

**Discussion by acount112** [community/community#45909] [New to GitHub] [answered]
### Select Topic Area

Question

### Body

My contribution is already showing up in my profile. I select my commit time in the Repository Insights contributor page box and it doesn't appear in the Repository Insights contributor page, Moreover, the Repositories Insights contributor page at the time I selected did not display more than 100
Is it my account Settings?

**Comment by acount112** [comment, marked as answer]
Sorry, I got the wrong account number

**Reply by mohsen12852** [reply]
> Sorry, I got the wrong account number
